src/ansys/aedt/core/generic/general_methods.py

```diff
@@ -626,10 +622,7 @@ def env_path(input_version):
     "C:/Program Files/ANSYSEM/ANSYSEM2021.2/Win64"
     """
     return os.getenv(
-        "ANSYSEM_ROOT{0}{1}".format(
-            get_version_and_release(input_version)[0], get_version_and_release(input_version)[1]
-        ),
-        "",
+        f"ANSYSEM_ROOT{get_version_and_release(input_version)[0]}{get_version_and_release(input_version)[1]}" "",
```

Comment:
```suggestion
        f"ANSYSEM_ROOT{get_version_and_release(input_version)[0]}{get_version_and_release(input_version)[1]}", ""
```
Typo here on getenv return value if key not found.